### PR TITLE
Initial trace4cats implementation

### PR DESCRIPTION
**Pull request by janstenpickle**
Adds wrappers for span completer and exporters that record various metrics

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+val Scala213 = "2.13.10"
+ThisBuild / crossScalaVersions := Seq("2.12.15", Scala213, "3.2.0")
+ThisBuild / scalaVersion := Scala213 // the default Scala
+
+lazy val root = tlCrossRootProject.aggregate(trace4Cats)
+
+lazy val trace4Cats = project
+  .in(file("trace4cats"))
+  .settings(
+    name := "prometheus4cats-trace4cats",
```
[the remark this answers is not in the document]
Comment (TimWSpence) [reply build.sbt:28]: Yeah I could be persuaded either way as well. I guess one advantage of `contrib` is that it makes it more obvious what modules are part of the "core" project and what are additional (as we have multiple core modules already)

**Review by TimWSpence — APPROVED** [review]
Nice!

File: build.sbt (hunk 1 above)
[the remark this answers is not in the document]
Comment (janstenpickle) [reply build.sbt:28]: I don't see why we can't break it out later as a repo called `prometheus4cats-contrib-trace4cats`, let's go with `contrib`

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other such characteristics.
+
+Everyone is expected to follow the [Scala Code of Conduct] when discussing the project on the available communication channels.
+
+
+## Moderation
+
+If you have any questions, concerns, or moderation requests, please contact a member of the project.
+
+- [Chris Jansen](mailto:andrew.valencik@gmail.com)
```
[the remark this answers is not in the document]
Comment (janstenpickle) [reply CODE_OF_CONDUCT.md:12]: lol no

**Pull request merged**